### ISSUE TITLE
DO: catalog and enrichment namespace

### ISSUE THREAD

**Issue by oleurud** [CartoDB/cartoframes#1018]
Right now, under cartoframes.data we have 
- `observatory` folder wit the catalog things
- `enrichment` folder with the enrichment things

I think we should:
- create `cartoframes.data.observatory.catalog` with all the catalog stuff
- create `cartoframes.data.observatory.enrichment` with all the enrichment stuff
- in the `cartoframes.data` __ init __ I would add:
```
catalog
enrichment_points
enrichment_polygons
```

How do you see it? @esloho @alejandrohall @alrocar

**Comment by alrocar** [comment]
LGTM!

**Comment by oleurud** [comment]
I think we need to tackle this one sooner than later because if you do it at the end we could have some problems with the references in docs or examples

**Comment by alrocar** [comment]
OK, let's focus on the download feature and we can priorize this next. I'm moving to the this week column.

**Comment by alrocar** [comment]
Let's use this issue to rename de `Dataset` class in the catalog to anything else to avoid collisions with the data Dataset.

Name TBD yet.

**Comment by oleurud** [comment]
Also to unify `_WORKING_PROJECT` constant in one place (the __ init __ fi ofle observatory?)

**Comment by cmongut** [comment]
We are going to maintain the observatory concept for now so let's move the enrichment part inside 🙂

**Comment by Jesus89** [comment]
Directory tree:

```
cartoframes/data/
├── clients
├── dataset
├── observatory
│   ├── catalog
│   └── enrichment
└── services
```

API:

```py
from cartoframes.data import Dataset
from cartoframes.data.clients import SQLClient, DataObsClient
from cartoframes.data.services import Geocoding, Isolines
from cartoframes.data.observatory import Catalog, Category, Country, CatalogDataset \
    Geography, Provider, Variable, enrich_points, enrich_polygons
```

Question: Do we want to expose everything at the `data` level?

**Comment by Jesus89** [comment]
Closed in #1119
